internal/button/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "module": "commonjs",
```

Comment:
As I understand we want to stop generating any CJS and exclusively output ESM. Please see PR here: https://github.com/namehash/nameguard/pull/268/files

Appreciate your advice.